www/js/metrics.js

```diff
@@ -0,0 +1,184 @@
+'use strict';
+
+angular.module('emission.main.metrics',['nvd3', 'emission.services'])
+
+.controller('MetricsCtrl', function($scope, $ionicActionSheet, CommHelper) {
+    $scope.options = {
+        chart: {
+            type: 'multiBarChart',
+            height: 450,
+            margin : {
+                top: 20,
+                right: 20,
+                bottom: 40,
+                left: 55
+            },
+            showValues: true,
+            stacked: false,
+            x: function(d){ return d[0]; },
+            y: function(d){ return d[1]; },
+            /*
+            average: function(d) { 
+                var vals = d.values.map(function(item){
+                    return item[1];
+                });
+                return d3.mean(vals);
+            },
+            */
+
+            color: d3.scale.category10().range(),
+            // duration: 300,
+            useInteractiveGuideline: true,
+            // clipVoronoi: false,
+
+            xAxis: {
+                axisLabel: 'Date',
+                tickFormat: function(d) {
+                    return d3.time.format('%y-%m-%d')(new Date(d * 1000))
+                },
+                showMaxMin: false,
+                staggerLabels: true
```

Comment:
Is there other way instead of stagger like make the font smaller or the month, day and year to only show day and month and show the year only once on the first label.